students/jiyeongkim/westagram/users/views.py

```diff
@@ -42,5 +43,23 @@ def post(self, request) :
 
             return JsonResponse({"message": "SUCCESS"}, status=201)
 
-        except:
-            return JsonResponse({"message": "KEY_ERROR"}, status=400)
\ No newline at end of file
+        except KeyError:
+            return JsonResponse({"message": "KEY_ERROR"}, status=400)
+
+class LogInView(View):
+    def post(self, request) :
+        try:
+            data          = json.loads(request.body)
+            email         = data['email']
+            password      = data['password']
+
+            if not User.objects.filter(email=email).exists():
+                return JsonResponse({"Message": "EMAIL_ERROR"}, status=400)
+            if not User.objects.filter(password=password).exists():
+                return JsonResponse({"Message": "PASSWORD_ERROR"}, status=400)
```

Comment:
1. 로그인의 경우 아이디가 틀렸는지, 비밀번호가 틀렸는지 모르게 하는 것이 좋기 때문에 에러 메시지를 동일하게 해주어야 합니다.
2. 보통 Json으로 반환 되는 key의 값은 소문자로 구성되어 있습니다.(스네이크 케이스)
3. 인증과 관련된 에러의 상태 코드로 401을 사용합니다.
4. 비밀번호를 검증 할 때는 비밀번호의 존재 유무를 판단하는 것이 아닌, 아이디와 비밀번호를 동시에 만족하는 데이터가 있는지 확인해야 합니다.

```suggestion
            if not User.objects.filter(email=email).exists():
                return JsonResponse({"message": "INVALIED_USER"}, status=401)
            
            if not User.objects.filter(email=email, password=password).exists():
                return JsonResponse({"message": "INVALIED_USER"}, status=401)
```